packages/openbridge-webcomponents/CHANGELOG.md

```
@@ -19,6 +19,12 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 - **storybook:** release storybook channels link ([#863](https://github.com/Ocean-Industries-Concept-Lab/openbridge-webcomponents/issues/863)) ([8c726a4](https://github.com/Ocean-Industries-Concept-Lab/openbridge-webcomponents/commit/8c726a424256cd497e043b1b76507c693ef244ef))
 - **wrapper:** update wrapper versions ([#732](https://github.com/Ocean-Industries-Concept-Lab/openbridge-webcomponents/issues/732)) ([120b03b](https://github.com/Ocean-Industries-Concept-Lab/openbridge-webcomponents/commit/120b03bb10db0a42e865e83ed5c7a75199fa0588)), closes [#397bf119](https://github.com/Ocean-Industries-Concept-Lab/openbridge-webcomponents/issues/397bf119) [#bbde8906](https://github.com/Ocean-Industries-Concept-Lab/openbridge-webcomponents/issues/bbde8906)
 
+# Changelog
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)